include/jemalloc/internal/jemalloc_internal_inlines_c.h

```diff
@@ -496,7 +496,7 @@ bool free_fastpath(void *ptr, size_t size, bool size_hint) {
         assert(tsd_fast(tsd) ||
             *tsd_thread_deallocated_next_event_fastp_get_unsafe(tsd) == 0);
 
-        emap_alloc_ctx_t alloc_ctx;
+        emap_alloc_ctx_t alloc_ctx = {0, 0, false};
```

Comment:
JEMALLOC_CC_SILENCE_INIT

Reply:
Modified, also modified `JEMALLOC_CC_SILENCE_INIT` a bit so that `-Wmissing-field-initializers` will not be triggered when the struct to init has multiple variables with different sizes.